pkg/operator/ceph/file/mds/mds.go

```diff
@@ -249,7 +245,9 @@ func (c *Cluster) upgradeMDS() error {
 		return errors.Wrap(err, "failed to setting allow_standby_replay to false")
 	}
 
-	// In Pacific, standby-replay daemons are stopped automatically. Older versions of Ceph require us to stop these daemons manually.
+	// In Pacific, standby-replay daemons are stopped automatically. Older versions of Ceph require
+	// us to stop these daemons manually.
+	// TODO: so why don't we have a version check?
```

Comment:
This looks like it's always failing replay daemons whether it's pacific or not. @jshen28 Do we need to check for Pacific or was this comment outdated?